tests/unit/states/archive_test.py

```diff
@@ -72,15 +67,9 @@ def test_extracted_tar(self):
                                                    'file.file_exists': mock_false,
                                                    'file.makedirs': mock_true,
                                                    'cmd.run_all': mock_run}):
-                    if HAS_PWD:
-                        running_as = pwd.getpwuid(os.getuid()).pw_name
```

Comment:
(git) 49d8bd1d  fix-tests [origin/fix-tests]
erik v(saltdev) ~/git/salt/fix-tests/tests                                  [17:48:43]
(⌐■_■) % python2 runtests.py -n unit.states.archive_test.ArchiveTest.test_extracted_tar
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 * Transplanting configuration files to '/tmp/salt-tests-tmpdir/config'
 * Current Directory: /home/erik/git/salt/fix-tests/tests
 * Test suite is running under PID 25106
 * Logging tests on /tmp/salt-runtests.log
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Starting unit.states.archive_test.ArchiveTest.test_extracted_tar Tests
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
.
----------------------------------------------------------------------
Ran 1 test in 0.004s

OK

===============================  Overall Tests Report  ================================
***  No Problems Found While Running Tests  *******************************************
=======================================================================================
OK (total=1, skipped=0, passed=1, failures=0, errors=0)
===============================  Overall Tests Report  ================================
(git) 49d8bd1d  fix-tests [origin/fix-tests]
erik v(saltdev) ~/git/salt/fix-tests/tests                                  [17:48:46]
(⌐■_■) % grep HAS_PWD unit/states/archive_test.py
(git) 49d8bd1d  fix-tests [origin/fix-tests]
erik v(saltdev) ~/git/salt/fix-tests/tests                                  [17:49:11]
（╯°□°）╯︵ ┻━┻ 1 % cat /etc/os-release
NAME="Arch Linux"
ID=arch
PRETTY_NAME="Arch Linux"
ANSI_COLOR="0;36"
HOME_URL="https://www.archlinux.org/"
SUPPORT_URL="https://bbs.archlinux.org/"
BUG_REPORT_URL="https://bugs.archlinux.org/"

(git) 49d8bd1d  fix-tests [origin/fix-tests]
erik v(saltdev) ~/git/salt/fix-tests/tests                                  [17:50:13]
(⌐■_■) % git describe
v2015.5.5-24-g49d8bd1